pkg/nsx/services/common/types.go

```diff
@@ -59,7 +59,7 @@ const (
 	LabelDefaultSubnetSet              string = "nsxoperator.vmware.com/default-subnetset-for"
```

Comment:
Why this one alone uses "nsxoperator.vmware.com"?

Reply:
I think this label we used it to indicate the default subnetset is created for VM or POD,
like
labels:
    nsxoperator.vmware.com/default-subnetset-for: VirtualMachine
Do we also need to change it:
    nsx.vmware.com/default-subnetset-for?

Reply:
If we follow the way to use nsxoperator.vmware.com for NSX Operator specific logic (not exposed to end users), I am fine with that too.

Reply:
Thanks.  We keep this specific label as it is for internal use.